app/src/plugins/launchDarkly.ts

```diff
@@ -34,10 +34,9 @@ export async function InitLdClient (): Promise<void> {
   const envKey: string = window['ldClientId']
 
   if (envKey) {
-    const user: LDUser = {
-      // since we have no user data yet, use a shared key temporarily
-      key: 'anonymous'
-    }
+    // since we have no user or org data yet, start with an anonymous user context
+    const context: LDContext = { kind: 'user', key: 'anonymous', anonymous: true }
```

Comment:
NB - I tested whether the initial call should also provide anonymous org (and device) contexts, and it didn't matter, so I simplified it to user only.